src/main/java/core/basesyntax/dao/FruitShopDaoCsvImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.storage.FruitShop;
+
+public class FruitShopDaoCsvImpl implements FruitShopDao {
+    private static final FruitShopDaoCsvImpl INSTANCE = new FruitShopDaoCsvImpl();
+
+    private FruitShopDaoCsvImpl() {
+    }
+
+    public static FruitShopDaoCsvImpl getInstance() {
+        return INSTANCE;
+    }
```

Comment:
why do you need this?
```suggestion
    private static final FruitShopDaoCsvImpl INSTANCE = new FruitShopDaoCsvImpl();

    private FruitShopDaoCsvImpl() {
    }

    public static FruitShopDaoCsvImpl getInstance() {
        return INSTANCE;
    }
```